docs/reference/connector/apis/update-connector-filtering-api.asciidoc

```diff
@@ -0,0 +1,186 @@
+[[update-connector-filtering-api]]
+=== Update connector filtering API
+
+preview::[]
+
+++++
+<titleabbrev>Update connector filtering</titleabbrev>
+++++
+
+Updates the `filtering` configuration of a connector.
```

Comment:
Should we link to [sync rules](https://www.elastic.co/guide/en/enterprise-search/current/sync-rules.html) overview here?

Reply:
Something like "Learn more about filtering in the `{enterprise-search-ref}/sync-rules.html[sync rules] documentation.`